calico.go

```diff
@@ -125,12 +125,19 @@ func cmdAdd(args *skel.CmdArgs) error {
 				return err
 			}
 
+			// Get labels
```

Comment:
I'd prefer a slightly more helpful comment here.

Even if it's "Get labels from Mesos if any are present"